recipes/cmake/3.x.x/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.15.7":
+    sha256: 71999d8a14c9b51708847371250a61533439a7331eb7702ac105cfb3cb1be54b
+    url: https://cmake.org/files/v3.15/cmake-3.15.7.tar.gz
```

Comment:
Minor thing: can you please put it somewhere lower in the list to keep it sorted? Same goes for the `config.yml`.

Reply:
It is sorted ascending, so it should stay as it is

Reply:
Ah oops, sorry.
You're correct. I was mistaken....